cmd/infra/aws/create_cli_role.go

```diff
@@ -0,0 +1,246 @@
+package aws
```

Comment:
I would love for this to be reviewed and merge after #https://github.com/openshift/hypershift/pull/3975